db/items.db

```diff
@@ -10,7 +10,7 @@ CREATE TABLE IF NOT EXISTS Categories (
     );
 CREATE TABLE IF NOT EXISTS Drafts(
     draft_id INTEGER PRIMARY KEY,
-    sort_index INTEGER NOT NULL,
+    sort_index INT NOT NULL AS (draft_id) STORED,
```

Comment:
Could you walk me through your thinking with using `AS (draft_id) STORED`? I'm not totally familiar with this usage and I'm curious how this implementation interacts with the `draft_id`.

Reply:
> Could you walk me through your thinking with using `AS (draft_id) STORED`? I'm not totally familiar with this usage and I'm curious how this implementation interacts with the `draft_id`.

The AS just generates the value of the sort_index to be the same as the draft_id upon creation of the item in the draft table. The STORED keyword has it being computed at the time of writing the row. I did this because the auto-incrementing can only be done on one column in sqlite3.

Reply:
Ah makes sense! So what I'm understanding is the initial value will be set to the same as `draft_id` but it will be able to be updated later as the user sorts the drafts. Perfect, thank you!